Feature/add opening times

markysoft opened the pull request:
Added reception and surgery opening times

st3v3nhunt commented on README.md:
```diff
@@ -44,7 +44,56 @@ The output JSON will be an arary of objects in the following format:
+                    "opens" : "15:00",
+                    "closes" : "19:00"
+                }
+            ...
```
Like it 👍

st3v3nhunt commented on app/lib/mappers/mapOpeningTimes.js:
```diff
@@ -0,0 +1,29 @@
+
+function getSessions(session) {
+  const timesSessions = session['s:timesSession'];
+  if (timesSessions.constructor === Array) {
```
Is `timesSessions` ever going to be `null`/`undefined`?

st3v3nhunt commented on app/lib/mappers/mapOpeningTimes.js:
```diff
@@ -0,0 +1,29 @@
+
+function mapOpeningTimes(rawTimes) {
+  const openingTimes = {};
+  rawTimes['s:daysOfWeek']['s:dayOfWeek'].forEach((dayOfWeek) => {
```
Are either `rawTimes['s:daysOfWeek']` or `['s:dayOfWeek']` ever going to be `null`/`undefined`?

st3v3nhunt commented on app/lib/mappers/mapOpeningTimes.js:
```diff
@@ -0,0 +1,29 @@
+function mapOpeningTimes(rawTimes) {
+  const openingTimes = {};
+  rawTimes['s:daysOfWeek']['s:dayOfWeek'].forEach((dayOfWeek) => {
+    openingTimes[dayOfWeek['s:dayName'].toLowerCase()] = getSessions(dayOfWeek['s:timesSessions']);
```
Will either `dayOfWeek['s:dayName']` or `dayOfWeek['s:timesSessions']` ever be `null`/`undefined`?

st3v3nhunt commented on test/resources/overview-no-opening-times.json:
```diff
@@ -0,0 +1,206 @@
+            }
+        }
+    }
+}
```
New line please

markysoft commented:
Added null checks, moved full json example out of README.mds into a separate file.
Add rate limiter to set overview scans to 5000 an hour. Reduced workers to 1.
Moved integration tests out of git hub and put them on travis' after_success
